### ISSUE TITLE
circleci (CLI) is not found in example

### ISSUE THREAD

**Issue by tnir** [circleci/circleci-docs#6748]
# Description
The example in parallelism shows "circleci (CLI) is not found".

This can be confirmed even outside of CircleCI:
```
$ docker run --rm -it circleci/buildpack-deps:trusty circleci
docker: Error response from daemon: OCI runtime create failed: runc create failed: unable to start container process: exec: "circleci": executable file not found in $PATH: unknown.
```

# Location
https://circleci.com/docs/2.0/configuration-reference/#parallelism

# Steps to Reproduce
- Copy, paste and run on CircleCI cloud:
```
jobs:
  build:
    docker:
      - image: buildpack-deps:trusty
        auth:
          username: mydockerhub-user
          password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
    environment:
      FOO: bar
    parallelism: 3
    resource_class: large
    working_directory: ~/my-app
    steps:
      - run: go test -v $(go list ./... | circleci tests split)
```

**Comment by tnir** [comment]
Will be resolved in #6749 in hours.

**Comment by rosieyohannan** [comment]
@tnir Hello! so the reason that the CLI is not being found using this example is because of the Docker image used for the executor. I am updating this example to use one of our convenience images which includes a subset of the CircleCI CLI. Thanks for pointing out this issue! :-D
